src/ispc.h

```diff
@@ -777,13 +777,16 @@ struct Globals {
     bool generateDebuggingSymbols;
 
     /** Require generation of DWARF of certain version (2, 3, 4). For
-        default version, this field is set to 0. */
+        default version, this field is set by deafaultDwarfVersion. */
     // Hint: to verify dwarf version in the object file, run on Linux:
     // readelf --debug-dump=info object.o | grep -A 2 'Compilation Unit @'
     // on Mac:
     // xcrun dwarfdump -r0 object.o
     int generateDWARFVersion;
 
+    /** Default DWARF version to be used. */
+    int defaultDWARFVersion;
```

Comment:
I think this default version should be implemented as a `constexpr`. Is there a use case where the default will change value?